src/main/java/org/kuali/kfs/pdp/businessobject/PaymentDetail.java

```diff
@@ -204,19 +216,13 @@ public void addNote(PaymentNoteText pnt) {
             pnt.setPaymentDetail(this);
             notes.add(pnt);
         } else {
-            LOG.warn("Did not add note to payment detail build from Document #: "+(!StringUtils.isBlank(custPaymentDocNbr) ? custPaymentDocNbr : "")+" because note was empty");
-        }
-    }
-
-    public void addNotes(List<PaymentNoteText> pnts) {
-        for(PaymentNoteText pnt : pnts) {
-            addNote(pnt);
+            LOG.warn("Did not add note to payment detail build from Document #: " + (!StringUtils.isBlank(custPaymentDocNbr) ? custPaymentDocNbr : "") + " because note was empty");
```

Comment:
we don't often use log warnings, should this be an error?

Reply:
This is how the line appears in the base PaymentDetail file, so I don't think we need to change the logging level in this case.

Reply:
OK, lets leave as is